cxx-qt-gen/src/generator/cpp/mod.rs

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::syntax::cpp::CppPair;
+
+/// Representation of the generated C++ code for a QObject
+pub struct GeneratedCpp {
```

Comment:
This isn't the generated C++ code, this is the data used to generate the C++ code, so I don't think this name fits. What could be a better name? CppData?

Reply:
It's generated from the perspective of, from the input of Rust code, this is the abstract representation of the generated C++ code. And only abstract from the point of view of "these things are methods" and "these things are sources". The actual strings inside them are full generated. The writer phase just takes this "generated" C++ code and performs a layout on it with parts that are static.

I'm not sure that CppData is a better name, as is this an input or output? is it a syntax CppData input or a parsed CppData abstract presentation or a generated CppData or a written out CppData header? :thinking: GeneratedAbstractCpp or GeneratedCppParts or GeneratedCppBlocks or something seem better? I think `GeneratedCppBlocks` is the best so far...

Reply:
Renamed to GeneratedCppBlocks